go-controller/pkg/ovn/master_test.go

```diff
@@ -666,13 +691,9 @@ var _ = Describe("Gateway Init Operations", func() {
 		app = cli.NewApp()
 		app.Name = "test"
 		app.Flags = config.Flags
-
-		stopChan = make(chan struct{})
 	})
 
 	AfterEach(func() {
-		close(stopChan)
-		f.Shutdown()
```

Comment:
@abhat also if we are creating a factory object, we need to call Shutdown() on it.

Reply:
deleted inadvertently. fixed it now.